pika/adapters/select_connection.py

```diff
@@ -367,11 +371,11 @@ def _process_fd_events(self, fd_event_map, write_only):
                 handler = self._fd_handlers[fileno]
                 handler(fileno, events, write_only=write_only)
 
-    def flush_outbound(self):
+    def _flush_outbound(self):
```

Comment:
@wjps, I think it would be better to simply delete this unnecessary method from SelectPoller. I believe that nothing is using it and I think that it was added anew in your recent PR.